integreat_cms/deepl_api/utils.py

```diff
@@ -0,0 +1,87 @@
+import deepl
+import os
+
+from django.contrib import messages
+
+
+class DeepLApi:
+    """
+    DeepL API to auto translate selected posts.
+    """
+
+    def check_availability(self, request, language_slug):
+        """
+        this function checks, if the selected language is supported by DeepL
+
+        :param request: request that was sent to the server
+        :type request: ~django.http.HttpRequest
+
+        :param language_slug: current language slug
+        :type language_slug: str
+
+        :return: true or false
+        :rtype: boolean
+        """
+        deepl_langs = [
+            "bg",
+            "cs",
+            "da",
+            "de",
+            "el",
+            "en",
+            "es",
+            "et",
+            "fi",
+            "fr",
+            "hu",
+            "it",
+            "ja",
+            "lt",
+            "lv",
+            "nl",
+            "pl",
+            "pt",
+            "ro",
+            "ru",
+            "sk",
+            "sl",
+            "sv",
+            "zh",
+        ]
+        if language_slug in deepl_langs:
+            return language_slug in deepl_langs
+        else:
+            messages.warning(
+                request,
+                _("This language isn't supported by DeepL. Please try another one"),
+            )
+
+    def get_default_language(self, region):
+        """
+        # zu implementieren am 21.1.
+        This function gets the default language of the cms
+        """
+        default_language = region.default_language.slug
+        print(default_language)
+
+    def get_post(self, poi_ids):
+        """
+        # zu implementieren am 21.1.
+        # nachdem alle Infos zum Beitrag übergeben werden,
+        # wird nur der tatsächliche Inhalt des Posts geholt
+        """
+        print(poi_ids)
```

Comment:
```suggestion
```
This method is probably also not necessary - you can just access `poi.content` to get the content of a POI without an additional function.